src/components/app/details/cicdHistory/types.tsx

```diff
@@ -29,6 +29,7 @@ export interface History {
     blobStorageEnabled?: boolean
     isArtifactUploaded?: boolean
     userApprovalMetadata?: UserApprovalMetadataType
+    IsVirtualEnvironment?: boolean
```

Comment:
IsVirtualEnvironment should be isVirtualEnvironment

Reply:
field is also capatalise, will do with BE changes